packages/cli/generation/ir-generator/src/converters/services/convertExampleEndpointCall.ts

```diff
@@ -514,6 +514,7 @@ function buildUrl({
             ...pathParams.servicePathParameters,
             ...pathParams.rootPathParameters
         ]) {
+            // TODO: should we URL encode the value?
```

Comment:
I think it makes sense to only do this on the frontend? That way if you just want to render the value somewhere where it doesn't make sense to url encode, you don't need to